app/assets/javascripts/utils/editable.js

```diff
@@ -0,0 +1,30 @@
+$(() => {
+  const readMode = $parent => {
+    $parent.find('.editable-cancel, .editable-save').remove();
```

Comment:
This naming seems a little risky; if we accidentally one of these `editable-save` or `editable-cancel` classes onto something in a React view, then we'd have jquery meddling with the view, right?

Maybe maybe it explicit that this is for rails forms? `rails_editable-cancel` and so on? (I'm not too experienced with jquery, so maybe this isn't something to worry about?)